data/lua/core/interface.lua

```diff
@@ -36,6 +37,49 @@ if wesnoth.kernel_type() == "Game Lua Kernel" then
 		moving_unit:to_map(to_x, to_y)
 		wesnoth.wml_actions.redraw{}
 	end
+	---@param u unit
+	---@param to_x integer
+	---@param to_y integer
+	---@param queued boolean
+	---@param scroll boolean
+	function wesnoth.interface.move_unit_fake_queue(u, to_x, to_y, queued, scroll)
+        if to_x and to_y and u then
+			if scroll == nil then
+				scroll = true
+			end
+			table.insert(queued_movements, { to_x = to_x, to_y = to_y, u = u, scroll = scroll })
+		end
+		if queued then
+			return
+		end
+
+		for _index, move in ipairs(queued_movements) do
+			local moving_unit = move.u
+			local from_x, from_y = moving_unit.x, moving_unit.y
+			local scroll = move.scroll
+			if scroll then
+				wesnoth.interface.scroll_to_hex(from_x, from_y)
+			end
+			to_x, to_y = wesnoth.paths.find_vacant_hex(move.to_x, move.to_y, moving_unit)
+			if to_x < from_x then
+				moving_unit.facing = "sw"
+			elseif to_x > from_x then
+				moving_unit.facing = "se"
+			end
+			moving_unit.hidden = true;
+            wesnoth.wml_actions.move_unit_fake {
```

Comment:
Just noting the CI check is still failing on the use of spaces for indentation here.